minecraft/protocol/packet/player_action.go

```diff
@@ -16,6 +16,9 @@ type PlayerAction struct {
 	// BlockPosition is the position of the target block, if the action with the ActionType set concerned a
 	// block. If that is not the case, the block position will be zero.
 	BlockPosition protocol.BlockPos
+	// ResultPosition is the resulting position of the target block. It is unclear what the difference between this
+	// field and BlockPosition is.
+	ResultPosition protocol.BlockPos
```

Comment:
I think I can explain what this is for: When a UseItemOn action is sent, the BlockPosition is the block clicked, but when a block is placed, this ResultPosition is the position at which the block will be placed.

Reply:
Would add that to the documentation.

Reply:
Added.